src/components/Modal/AddMailModal.tsx

```diff
@@ -53,6 +54,19 @@ export const AddMailModal = (props: IAddMailModalProps) => {
   const [newMail, setNewMail] = useState("");
   const [confirmMail, setConfirmMail] = useState("");
 
+  /*
+   * Boolean variable to check if a work email different from spid one is already set
+   * */
+  const isWorkMailNotSet = !props.workMail || props.workMail === props.spidMail;
+
+  /*
+   * If a work mail different from spid one is already set, user in changing work email
+   * otherwise is adding work mail
```

Comment:
```
// If a work email address is already set and differs from the SPID email, 
// then the user is changing his work email address,  otherwise he's adding it

```

apart from the typo, this kind of comments should follow the C++ convention,
reserve the C format for methods descriptions:

```
/**
  *   <description here>
  */
function f()
```